README.md

```diff
@@ -1,22 +1,37 @@
-# A Discord Rich Presence addon for Guild Wars 2.
-
-https://gw2rpc.info
-
-## Features:
-* Displaying off:
-   + current map, as well as closes point of interest
-   + current raid or fractal boss
-   + character name, race and profession (elite spec)
-   + commander icon if player is currently commanding a squad
-   + active guild tag (needs API key)
-   + time spent on map
+<p align="center"><img alt="Guild Wars 2 RPC" src="https://media.discordapp.net/attachments/908219512271286282/965798895433170954/GW2.png?width=473&height=473" height="76"></p></img>
+
+<h1 align="center">Guild Wars 2 RPC</h1>
+<p align="center">A Discord Rich Presence addon for Guild Wars 2.</p>
+
+<a href="https://gw2rpc.info"><p align="center">Go to website</a>・<a href="https://github.com/Maselkov/GW2RPC/releases">Download Latest</a>・<a href="https://github.com/Maselkov?tab=repositories">Other Projects...</p></a>
+
+---
+<p align="center">You need install <img src="https://api.iconify.design/bi:discord.svg?color=%23f9f9f9" height="12"> <b><a href="https://discord.com/download">Discord Desktop version</b></a> to run the rich presence.</p>
```

Comment:
Small grammar: "You need **to** install"

Reply:
sorry my english is bad :b